pkg/actors/internal/placement.go

```diff
@@ -112,6 +113,18 @@ func NewActorPlacement(
 	}
 }
 
+// Force send an update to placement service (i.e late registration of internal actors)
```

Comment:
Q: Which part of the implementation forces the update to the placement service?

Reply:
I will change that, it doesn't actually force an update, the update will happen with the heartbeat